packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -39,6 +41,18 @@ const rtl = css`
     }
 `;
 
+const neutralStealthHoverOverHoverBehavior = cssCustomPropertyBehaviorFactory(
```

Comment:
Fair point. I tweaked those names a bit as I was thinking of aligning with standard recipes, but something semantic would be easier to understand since they are local. `expandCollapseHover' maybe.